security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+Security is at the heart of a healthy and dependable software ecosystem and therefore also at the heart of the SSWG. This page intends to give clear and easy to follow information to users and developers of packages listed on the SSWG's package index.
+
+If you find any information to be missing, unclear, or misleading, please contact any member of the SSWG or create a post in the [server area](https://forums.swift.org/c/server/serverdev/14) of the Swift Forums.
+
+## I found a security vulnerability, what should I do?
+
+If you found a security vulnerability, first of all thank you for caring about our ecosystem. We rely on the community to keeping us all as safe and secure as we can (at least when it comes to software).
+
+The SSWG only takes a secondary role in the security process. The SSWG makes sure that listed projects do have a security process and that they follow their own process but the SSWG is typically not the first point of contact. Please don't read this as sending you away though, if at any point in the process you feel like you don't know where to report, you don't get a response, or face any other issues, please do contact the SSWG at [sswg-security-reports@forums.swift.org](mailto:sswg-security-reports@forums.swift.org) (this can only be seen by the SSWG members and the Swift core team).
+
```

Comment:
I think it may be better to switch these paragraphs so you point reporters to individual packages first, then the fallback is communicating with the SSWG directly. I also think this section would benefit from being much more concise.

Reply:
thanks, done. Better?